schemas/system/Subdivision.schema

```diff
@@ -1,4 +1,4 @@
 System({
   Name: { domain: 'Nomen', display: true, required: true, unique: true },
-  Parent: Parent('Division')
+  Parent: Parent('Subdivision')
```

Comment:
Does using `'Parent'` mean that it is required to have a one-to-many link with the specified category for every object, and if not, how to specify that such link is required?

Reply:
`Parent` gives us recursive one-to-many, so I think we can change it to: `Parent: Master('Subdivision')` So we need no special decorator for recursion.

Reply:
In this case, it is recursive, but I was asking about a general case (when the parent is a different category) if this field has to be filled so that there's always a link for each object.

Also, since this one is recursive, doesn't that mean that the top one `Subdivision` may have no parent `Subdivision` or are they always linked in a circular manner?

Reply:
We will use: `Parent: { category: 'Subdevision' }`